<launch>

    <node pkg="stereo_vo" type="stereo_vo" name="vo" output="log" 
    args="/home/cooper/ws/uav_ws/src/stereo_vo_estimator/config/realsense_2/realsense.yaml">
    </node>

    <!-- <node pkg="tag_detector" type="tag_detector" name="tag_detector" output="screen">
        <remap from="~image_raw" to="/flattener/flatImage_0"/>
        <param name="cam_cal_file" type="string" value="$(find tag_detector)/config/pt_grey.yml"/>
        <param name="board_config_file" type="string" value="$(find tag_detector)/config/a.yml"/>
    </node> -->

    <node pkg="aug_ekf" type="aug_ekf" name="aug_ekf" output="screen">
        <remap from="~imu" to="/mavros/imu/data"/>
        <remap from="~tag_odom" to="/tag_detector/odom_ref_"/>

        <param name="aug_ekf/ng" type="double"  value="1.0"/>
        <param name="aug_ekf/na" type="double"  value="1.0"/>
        <param name="aug_ekf/nbg" type="double" value="0.1"/>
        <param name="aug_ekf/nba" type="double" value="0.1"/>
        <param name="aug_ekf/pnp_p" type="double" value="0.5"/>
        <param name="aug_ekf/pnp_q" type="double" value="0.5"/>
        <param name="aug_ekf/vo_pos" type="double" value="0.1"/>
        <param name="aug_ekf/vo_rot" type="double" value="0.1"/>
        <param name="aug_ekf/simple_bag" type="bool" value="false"/>
    </node> 

</launch>
